examples/default/src/screens/CrashReportingScreen.tsx

```diff
@@ -46,6 +50,48 @@ export const CrashReportingScreen: React.FC = () => {
     }
   }
 
+  const styles = StyleSheet.create({
+    inputWrapper: {
+      padding: 4,
+      flex: 1,
+    },
+
+    formContainer: {
+      flexDirection: 'row',
+      alignItems: 'stretch',
+    },
+  });
+  const [userAttributeKey, setUserAttributeKey] = useState('');
+  const [userAttributeValue, setUserAttributeValue] = useState('');
+  const [crashNameValue, setCrashNameValue] = useState('');
+  const [crashFingerprint, setCrashFingerprint] = useState('');
+  const [crashLevelValue, setCrashLevelValue] = useState<NonFatalErrorType>(
+    NonFatalErrorType.error,
+  );
+
+  function sendCrash() {
+    try {
+      const error = Error(crashNameValue);
```

Comment:
This should be `new Error` I think.

```suggestion
      const error = new Error(crashNameValue);
```